CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [unreleased]
 
 ### Added
+- **PODAAC-4852**
+  - Add HOSS jupyter test notebook
 ### Deprecated
 ### Removed
 ### Fixed
```

Comment:
Can you add a README.md with how to run it? Can you add an example how to run the notebook with an example input collection that works?

Reply:
Updated readme, the notebook has a default podaac collection in ops that the notebook runs with.